src/java/htsjdk/samtools/liftover/LiftOver.java

```diff
@@ -139,6 +143,29 @@ public Interval liftOver(final Interval interval, final double liftOverMinMatch)
         return ret;
     }
 
+    /**
+     * @return the set of destination contigs for each source contig in the chains file.
+     */
+    public Map<String, Set<String>> getContigMap() {
```

Comment:
I would actually like to change this to be static rather than keeping a member to encourage folks to only call this only once.  Consider genomes that both have many chain entries and many contigs.